jit/src/instructions.rs

```diff
@@ -403,7 +410,62 @@ impl<'a, 'b> FunctionCompiler<'a, 'b> {
                 // TODO: block support
                 Ok(())
             }
-            _ => Err(JitCompileError::NotSupported),
+            Instruction::LoadGlobal(idx) => {
+                let name = &names[*idx as usize];
+                if name.as_ref() == "print" {
+                    self.stack.push(JitValue {
+                        val: self.builder.ins().iconst(types::I8, 0), // Not used
+                        ty: JitType::PrintFunction,
+                    });
+                    Ok(())
+                } else {
+                    Err(JitCompileError::NotSupported)
+                }
+            }
+            Instruction::CallFunctionPositional { nargs } => {
+                if nargs != &1 {
+                    return Err(JitCompileError::NotSupported);
+                }
+
+                let arg1 = self.stack.pop().ok_or(JitCompileError::BadBytecode)?;
+                let function = self.stack.pop().ok_or(JitCompileError::BadBytecode)?;
+
+                match (arg1.ty, function.ty) {
+                    (JitType::Int, JitType::PrintFunction) => {
```

Comment:
I generally haven't touched on the `jit` folder and have no real experience with cranelift. Is my understanding correct that what this PR allows is a subcase of the Python print, i.e: `print(some_int_value)`?

Reply:
Exactly. `print` with exactly one int argument works.